int32 degree
int64 traj_id
time start_time

float64[] knots
geometry_msgs/Point[] ctrl_pts

float64[] yaw_pts
float64 yaw_dt

